lib/elixir/lib/string.ex

```diff
@@ -1504,6 +1504,24 @@ defmodule String do
       iex> String.next_codepoint("olá")
       {"o", "lá"}
 
+  Note, however, that getting the next codepoint via pattern
+  matching is generally preferred. For example
+
+  ```elixir
+  def process(<<cp::utf8, rest::binary>>) do
+    # ...
+  end
+  ```
```

Comment:
A style nitpick: we use the 4-space indentation for code blocks instead of the fences.

Reply:
Oh, I just read the end of your comment. Yeah, I think using the indentation would be better. We have those more complex examples indented like that in other parts of the codebase as well.

Reply:
Aaah, I did a project search and found [this occurrence](https://github.com/elixir-lang/elixir/blob/master/lib/mix/lib/mix/tasks/new.ex#L243-L248).

Let me revise it.

Reply:
Up!